pkg/operator/credentialsrequest/credentialsrequest_controller.go

```diff
@@ -233,6 +233,46 @@ func add(mgr, adminMgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	// allCredRequestsMapFn simply looks up all CredentialsRequests and requests they be reconciled.
+	allCredRequestsMapFn := handler.TypedEnqueueRequestsFromMapFunc(func(ctx context.Context, a *configv1.Infrastructure) []reconcile.Request {
```

Comment:
Please prefix the variable name with the type... `instrastructure.AllCredRequestsMapFn` to avoid clashes.

Reply:
Added!!